workflow/controller/opeartor_data_test.go

```diff
@@ -0,0 +1,247 @@
+package controller
```

Comment:
Spotted a typo in the file name - should be "operator_data_test.go"

Reply:
Thanks, changed